ingestion/create_deploy.sh

```diff
@@ -1,17 +1,25 @@
 #!/bin/bash
 
-echo "RALLY_API_URL: $RALLY_API_URL"
-echo "RALLY_WORKSPACE_OID: $RALLY_WORKSPACE_OID"
-echo "DEPLOY_COMPONENT_NAME: $DEPLOY_COMPONENT_NAME"
-echo "DEPLOY_BUILD_ID: $DEPLOY_BUILD_ID"
-echo "DEPLOY_START_TIME: $DEPLOY_START_TIME"
-echo "DEPLOY_END_TIME: $DEPLOY_END_TIME"
-echo "DEPLOY_IS_SUCCESSFUL: $DEPLOY_IS_SUCCESSFUL"
-echo "DEPLOY_MAIN_REVISION: $DEPLOY_MAIN_REVISION"
-echo "PREVIOUS_SUCCESS_BUILD_COMMIT: $PREVIOUS_SUCCESS_BUILD_COMMIT"
-echo "CURRENT_BUILD_COMMIT: $CURRENT_BUILD_COMMIT"
-echo "GIT_REPO_LOC: $GIT_REPO_LOC"
-
+VERSION=1
+NAME=$(basename $0)
+AUTHOR="Iron Will Beard"
+
+SOURCE_SYSTEM_META_DATA="{ \\\"name\\\":\\\"${NAME}\\\",\\\"version\\\":${VERSION},\\\"author\\\":\\\"${AUTHOR}\\\" }"
+
+echo "RALLY_API_URL: ${RALLY_API_URL}"
+echo "RALLY_WORKSPACE_OID: ${RALLY_WORKSPACE_OID}"
+echo "DEPLOY_COMPONENT_NAME: ${DEPLOY_COMPONENT_NAME}"
+echo "DEPLOY_BUILD_ID: ${DEPLOY_BUILD_ID}"
+echo "DEPLOY_START_TIME: ${DEPLOY_START_TIME}"
+echo "DEPLOY_END_TIME: ${DEPLOY_END_TIME}"
+echo "DEPLOY_IS_SUCCESSFUL: ${DEPLOY_IS_SUCCESSFUL}"
+echo "PREVIOUS_SUCCESS_BUILD_COMMIT: ${PREVIOUS_SUCCESS_BUILD_COMMIT}"
+echo "CURRENT_BUILD_COMMIT: ${CURRENT_BUILD_COMMIT}"
+echo "GIT_REPO_LOC: ${GIT_REPO_LOC}"
+echo "DEPLOY_BUILD_URL: ${DEPLOY_BUILD_URL}"
+echo "COMMIT_OVERRIDE: ${COMMIT_OVERRIDE}"
+#temporary
+echo "API_KEY: ${RALLY_API_KEY}"
```

Comment:
Probably remove this

Reply:
yeah, the #temporary was my reminder to do that...and still I forgot :-)